### PR TITLE
Begining of portfolio mobile version

### PR DESCRIPTION
**Pull request by GintsM**
Created Portfolio's first page, mobile version. Able to use as web, must change the background image for that.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <title>GintsMisins Portfolio</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header class="title-row">
+    <h4 class="header_content" id="logo">GintsMisins</h4>
```
Comment (karvel-code) [inline index.html:13]: - [ ] Please ensure that the logo is a link. ie, it should redirect the user to the homepage. You can achieve this by wrapping it inside the `<a>` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <p class="header_content" id="nav">≡</p>
```
Comment (karvel-code) [inline index.html:14]: - [ ] Please ensure that you wrap the hamburger icon inside the `<nav>` tag. Its is also advisable to use the hamburger icon available on f[ontawsome](https://fontawesome.com/).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  </header>
```
Comment (karvel-code) [inline index.html:15]: - [ ] Please ensure you use the font family used in the design for this section which is Poppins as shown in the design.

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @GintsM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Git flow has been adhered to.
- Your code looks neat and professional.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 3 above)
Comment (GintsM) [reply index.html:15]: Thanks for the advice but the linter showed some issues with family font, so I went to check for that inhttps://stylelint.io/user-guide/rules/list/font-family-no-missing-generic-family-keyword/ , and took one of mentioned font
 maybe I didn't get it right
I will try to change it back.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <title><a href="">GintsMisins Portfolio</a></title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+</head>
+<body>
+  <header class="title-row">
+    <h4 class="header_content" id="logo"><a href="">GintsMisins</a></h4>
```
Comment (SajjadAhmad14) [inline index.html:14]: [ OPTIONAL ] Please remove the underline decoration from the logo. You can achieve it by `text-decoration: none`.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Gints Mišins, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL
- [ ] Please see inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**